apps/codeforafrica/src/pages/api/admin/config.js

```diff
@@ -1,7 +1,9 @@
 module.exports = {
   backend: {
-    name: "git-gateway",
+    name: "github",
     branch: "main",
+    repo: "CodeForAfrica/ui",
```

Comment:
Can't we set these up in the [`config.yml.page`](https://github.com/CodeForAfrica/ui/blob/b96a2604e2f8393f5246522c69580272e12bc1fb/apps/codeforafrica/src/pages/api/admin/config.yml.page.js#L9-L19) for production use only i.e. default should still be local development

Reply:
With this setup, the default is still `local` just that we don't have to use the `git-gateway` backend

Reply:
Cool, cool... I think the setup was copied from the [docs](https://www.netlifycms.org/docs/beta-features/#working-with-a-local-git-repository) directly. e.g. not sure if we need `proxy_url` either.